ui/src/app/model/pipeline.model.ts

```diff
@@ -15,13 +15,14 @@ export class PipelineStatus {
     static SKIPPED = 'Skipped';
     static NEVER_BUILT = 'Never Built';
     static STOPPED = 'Stopped';
+    static PENDING = 'Pending';
```

Comment:
check in cli monitoring view if it's good with new status

Reply:
New status must not be in cli monitoring